modules/ossm-federation-config-meshPeer.adoc

```diff
@@ -0,0 +1,111 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-task_{context}"]
+= Joining a federated mesh
+
+You declare the federation between two meshes by creating a `ServiceMeshPeer` resource. This resource defines the federation between two meshes, and you use it to configure discovery for the peer mesh, access to the peer mesh, and certificates used to validate the other mesh’s clients.
+
+//Insert ServiceMeshPeer diagram here
+
+Configuration in the `ServiceMeshPeer` resource includes the following:
+
+* The address of the other mesh’s ingress gateway, which is used for discovery and service requests.
+* The names of the local ingress and egress gateways that is used for interactions with the specified peer mesh.
+* The client ID used by the other mesh when sending requests to this mesh.
+* The trust domain used by the other mesh.
+* The name of a `ConfigMap` containing a root certificate that is used to validate client certificates in the trust domain used by the other mesh.
+
+Meshes are federated on a one-to-one basis, so each pair of peers requires a `ServiceMeshPeer` file specifying the federation connection to the other service mesh. For example, federating three meshes named `red`, `blue`, and `green` would require six `ServiceMeshPeer` files.
```

Comment:
s/file/resource/

also: each pair of peers requires a *pair* of ServiceMeshPeer resources

I'd also start with a basic example containing only two meshes and only then expand it with an additional mesh. Right now, it starts of talking about two meshes, but the example then jumps to three. This could be confusing to the reader.